client/src/panel.tsx

```diff
@@ -258,6 +267,11 @@ export const Panel: React.FC<IsDropTargetProps> = ({ isDropTarget }) => {
           {title}
         </span>
       </Navigation>
+      {panelRef.current && (
+        <React.Fragment>
+          <Tour panelElement={panelRef.current} />
+        </React.Fragment>
```

Comment:
(minor) `React.Fragment` wrapper can be removed here
```suggestion
        <Tour panelElement={panelRef.current} />
```